Woof/WoofTests/ViewModel/LoginViewModelTests/LoginViewModelSetUserRoleTests.swift

```diff
@@ -0,0 +1,39 @@
+import XCTest
+
+final class LoginViewModelSetUserRoleTests: XCTestCase {
+    private var userPreferencesStorage = KeyValueStorage.testPreferencesStorage
+    private let testLoginViewModel = LoginViewModel()
+
+    override func setUp() {
+        super.setUp()
+        userPreferencesStorage.deleteData(for: KeyValueStorage.Key.userPreferences)
+    }
+
+    func testMethodExistsInAPI() {
+        testLoginViewModel.set(.sitter)
+    }
+
+    func testSetsEveryPossibleRole() {
+        Role.allCases.forEach {
+            // Given When
```

Comment:
nit:


```suggestion
            // Given // When
```

to be consistent with other test cases.

Reply:
Understood